libs/librepcb/common/application.h

```diff
@@ -90,6 +91,7 @@ class Application final : public QApplication
     private: // Data
         Version mAppVersion;
         QString mGitVersion;
+        QString mBuildDate;
```

Comment:
As it is a date/time, I would use QDateTime for that.

Reply:
Done